composer.json

```diff
@@ -23,18 +23,18 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
         "symfony/framework-bundle": "^3.4 || ^4.0 || ^5.0",
-        "laminas/laminas-diagnostics": "^1.5"
+        "laminas/laminas-diagnostics": "^1.8"
```

Comment:
Can you leave as is. We need to support PHP 7.1 for now... This will fix the test failure.